examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml

```diff
@@ -0,0 +1,95 @@
+# Punctuation and capitalization with pretrained BERT models
+
+pl:
+  trainer:
+    gpus: 1 # the number of gpus, 0 for CPU
+    num_nodes: 1
+    max_epochs: 3
+    max_steps: null # precedence over max_epochs
+    accumulate_grad_batches: 1 # accumulates grads every k batches
+    amp_level: O0 # O1/O2 for mixed precision
+    distributed_backend: dpp
+
+model:
+  data_dir: ??? # /path/to/data
+  class_balancing: null # or weighted_loss
+
+  train_ds:
+    prefix: train
+    max_seq_length: 128
+    pad_label: 'O'
+    num_samples: -1
+    ignore_extra_tokens: false
+    ignore_start_end: false
+    overwrite_processed_files: false
+    shuffle: true
+    batch_size: 2
+    num_workers:  2
+    pin_memory: false
+    drop_last: false
+
+  validation_ds:
+    prefix: dev
+    max_seq_length: 128
```

Comment:
max_seq_length, pad_label, ignore_extra_tokens, ignore_start_end are duplicated in both data loaders config. Is there a better way to have them in one place? For instance, how about moving them into the language_model section? They can be somehow related? Or having a separate section for them like preprocessing or dataset section?

Reply:
@ericharper but let me leave this as is for now, I followed text_classification config example, if we decided on a different config for nlp, let's do this in a separate PR